editor/src/components/canvas/canvas-strategies/absolute-reparent-to-flex-strategy.spec.browser2.tsx

```diff
@@ -357,7 +357,7 @@ describe('Absolute Reparent To Flex Strategy', () => {
     await renderResult.getDispatchFollowUpActionsFinished()
 
     const dragDelta = windowPoint({
-      x: firstFlexChildEnd.x - absoluteChildCenter.x + 50,
+      x: firstFlexChildEnd.x - absoluteChildCenter.x,
```

Comment:
why was this change needed?

Reply:
This was dragging over the centre of an element that is now a valid parent, so now it is instead being dragged just before it